Still no luck.

```
Downloading https://files.pythonhosted.org/packages/c5/39/4da7c2dbc4f023fba5fb2325febcadf0d0ce0efdc8
bd12083a0f65d20653/python-dateutil-2.7.2.tar.gz (298kB)", "  Downloading from URL https://files.pythonhosted.org/packages/c5/39/4da7c2dbc4f023fba5fb2325febcad
f0d0ce0efdc8bd12083a0f65d20653/python-dateutil-2.7.2.tar.gz#sha256=9d8074be4c993fbe4947878ce593052f71dac82932a677d49194d8ce9778002e (from https://pypi.org/sim
ple/python-dateutil/)", "  Running setup.py (path:/tmp/pip-build-knaads8k/python-dateutil/setup.py) egg_info for package python-dateutil", "    Running comman
d python setup.py egg_info", "    WARNING: The wheel package is not available.", "    Traceback (most recent call last):", "      File \"<string>\", line 1, i
n <module>", "      File \"/tmp/pip-build-knaads8k/python-dateutil/setup.py\", line 86, in <module>", "        \"test\": Unsupported", "      File \"/tmp/secu
redrop-app-code_1.9.0~rc1+xenial_amd64/debian/securedrop-app-code/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/setuptools/__init__.py\", line 144
, in setup", "        return distutils.core.setup(**attrs)", "      File \"/usr/lib/python3.5/distutils/core.py\", line 108, in setup", "        _setup_distri
bution = dist = klass(attrs)", "      File \"/tmp/securedrop-app-code_1.9.0~rc1+xenial_amd64/debian/securedrop-app-code/opt/venvs/securedrop-app-code/lib/pyth
on3.5/site-packages/setuptools/dist.py\", line 425, in __init__", "        k: v for k, v in attrs.items()", "      File \"/usr/lib/python3.5/distutils/dist.py
\", line 281, in __init__", "        self.finalize_options()", "      File \"/tmp/securedrop-app-code_1.9.0~rc1+xenial_amd64/debian/securedrop-app-code/opt/ve
nvs/securedrop-app-code/lib/python3.5/site-packages/setuptools/dist.py\", line 706, in finalize_options", "        ep.load()(self)", "      File \"/tmp/secure
drop-app-code_1.9.0~rc1+xenial_amd64/debian/securedrop-app-code/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/pkg_resources/__init__.py\", line 22
29, in load", "        return self.resolve()", "      File \"/tmp/securedrop-app-code_1.9.0~rc1+xenial_amd64/debian/securedrop-app-code/opt/venvs/securedrop-a
pp-code/lib/python3.5/site-packages/pkg_resources/__init__.py\", line 2235, in resolve", "        module = __import__(self.module_name, fromlist=['__name__'],
 level=0)", "      File \"/tmp/pip-build-knaads8k/python-dateutil/.eggs/setuptools_scm-6.0.1-py3.5.egg/setuptools_scm/__init__.py\", line 15, in <module>", " 
       from .version import format_version, meta", "      File \"/tmp/pip-build-knaads8k/python-dateutil/.eggs/setuptools_scm-6.0.1-py3.5.egg/setuptools_scm/v
ersion.py\", line 42", "        trace(f\"tag '{tag}' parsed to {result}\")", "                                              ^", "    SyntaxError: invalid synt
ax", "Cleaning up...", "
```

It is getting `setuptools_scm==6.0.1` for dateutil `/tmp/pip-build-knaads8k/python-dateutil/.eggs/setuptools_scm-6.0.1-py3.5.egg/setuptools_scm/version.py`.

--- a/install_files/securedrop-app-code/debian/rules
+++ b/install_files/securedrop-app-code/debian/rules
@@ -28,6 +28,7 @@ override_dh_virtualenv:
 	dh_virtualenv \
 		--python=/usr/bin/python3 \
 		--builtin-venv \
+		--preinstall setuptools-scm==5.0.2 \
 		--extra-pip-arg "--verbose" \
 		--extra-pip-arg "--no-deps" \
 		--extra-pip-arg "--no-binary=:all:" \

